atest/testdata/parsing/suite_names/child_tests01/child0102.robot

```diff
@@ -0,0 +1,13 @@
+*** Settings ***
+Documentation       Suite without custom name but with custom name in init
+
+Suite Name          Child 0102 Suite
+
+
+*** Test Cases ***
+Verify That Suite Name is not a file like name
+    Should Not Be Equal    ${SUITE_NAME}    Robot.Parsing.Parent Init Suite.Child Tests01.Child0101
+    Should Not Be Equal    ${SUITE_NAME}    Robot.Parsing.Parent Init Suite.Child Suite01 Name.Child0101
```

Comment:
Also this test is unnecessary. Removing it and just leaving `Verify Suite Name` makes the test simpler. Same comment applies to latter suites with similar structure.

The empty line between settings is unnecessary as well.